### PR TITLE
Deploy button do not work for all platforms 

### LINKED ISSUE #1001: Deployment buttons do not work for all platforms everywhere
The following two have issues:

* Heroku a) works from the Readme file in the repo, but does not work from dev.loklak.org
* Deploy to Docker does not work on Readme or on dev.loklak.org

### PR DESCRIPTION
**Pull request by AnaghHegde**
Fixes #1001

### CODE REVIEW

**Comment by AnaghHegde** [comment]
Someone change this in [dev.asksusi.com](dev.asksusi.com) also.  
- For docker add somethoing like this `https://cloud.docker.com/stack/deploy/?https://github.com/fossasia/susi_server` along with changes done above.  

- For heroku we need to remove that option (as there is no information regarding Adding the **Deploy to Heroku Cloud** button anywhere else and they also mentioned that The basic requirements for creating a button are that your app has a valid app.json file in its root directory, and that the app source code is hosted in a GitHub repository. For more info check [this](https://devcenter.heroku.com/articles/heroku-button#requirements) ).

File: README.rst [hunk 1]
```diff
@@ -157,6 +157,9 @@ change this if you want to. This is how connected peers work:
   back-end. You can create your own message-sharing network easily.
 
 How do I install loklak: Download, Build, Run
+
+> **Note**: You must be logged in to Docker Cloud for the button to work correctly. If you are not logged in, you’ll see a 404 error  instead.
```
Comment (yukiisbored) [inline README.rst]: Please use the `note` directive.

```rst
.. note::

   You must be logged in to Docker Cloud for the button to work correctly.
   If you are not logged in, you’ll see a 404 error  instead.

```

**Comment by AnaghHegde** [comment]
@yukiisbored OK i will make those changes.

**Comment by jig08** [comment]
@AnaghHegde updates on this pls ?

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**